packages/cmrjs/src/cmr-utils.js

```diff
@@ -1221,6 +1284,81 @@ async function getGranuleTemporalInfo(granule) {
   return {};
 }
 
+/**
+ * Search recursively for the location of a sub-object in the cmr object
+ *
+ * @param {object} cmrObject - cmr metadata object
+ * @param {string} attributePath - subObject path to seek
+ * @returns {string | null}
+*/
+const findCollectionAttributePath = (cmrObject, attributePath) => {
```

Comment:
Is there a reason we need to recurse through the entire object?   Is collection not a well defined key in the CMR spec?

Reply:
Also.... if it *isn't* a well defined location in the spec, are we certain there can't be duplicates/etc?

Reply:
This thread was resolved via a conversation on 2/7 - CMR spec *does* define the location, recursion isn't needed.